src/main/java/com/training/web/basePages/FlipkartBasePage.java

```diff
@@ -0,0 +1,9 @@
+package com.training.web.basePages;
+
+import org.openqa.selenium.By;
+
+public class FlipkartBasePage {
+    public By loaderIcon = By.xpath("//div[@class='_2YsvKq _3bgaUQ']/*[name()='svg']");
+    public By cartIcon = By.xpath("//*[text()='Cart']");
```

Comment:
Follow the java rules, make these variables private and have methods which uses these xpaths and use it in your pages.